petab/problem.py

```diff
@@ -347,66 +347,111 @@ def get_noise_distributions(self):
         return measurements.get_noise_distributions(
             measurement_df=self.measurement_df)
 
+    def _to_mask(v: List, free: bool = True, fixed: bool = True):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method should have "self" as first argument](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4968439)